harness/determined/profiler.py

```diff
@@ -505,20 +510,25 @@ def measure(self, batch_idx):
             handle = pynvml.nvmlDeviceGetHandleByIndex(i)
             try:
                 util = pynvml.nvmlDeviceGetUtilizationRates(handle)
-                gpu_util = util.gpu
-
-            except pynvml.NVMLError as err:
-                # TODO: Log this error
-                continue
-            measurements[handle] = Measurement(timestamp, batch_idx, gpu_util)
+                measurements[handle] = Measurement(timestamp, batch_idx, util.gpu)
+            except pynvml.NVMLError as e:
+                logging.warning(f"failed to sample GPU utilization: {e}")
```

Comment:
I think we need more informative logs. This will just show up in the user's logs without any context right?

Reply:
yeah i'll throw a gpu id in, sound good?